decidim-generators/lib/decidim/generators/app_templates/initializer.rb

```diff
@@ -1,30 +1,34 @@
 # frozen_string_literal: true
 
+# most of this configurations can be handled through the environment variables.
+# Please refer to: https://docs.decidim.org/en/develop/configure/environment_variables
+
 Decidim.configure do |config|
   # The name of the application
-  config.application_name = Rails.application.secrets.decidim[:application_name]
+  config.application_name = Decidim::Env.new("DECIDIM_APPLICATION_NAME", "My Application Name").to_s
```

Comment:
This should be defined in the config_accessor itself: https://github.com/decidim/decidim/blob/c889af128f7673f1c4d3619c30599cc5b8c801ad/decidim-core/lib/decidim/core.rb#L232  (The same for the mailer_sender and the rest of the things in this file)

Reply:
fixed in 22115a5511